kusto/ingest/internal/filesystem/filesystem.go

```diff
@@ -53,16 +53,28 @@ type Ingestion struct {
 
 	uploadBlobStream uploadBlobStream
 	uploadBlobFile   uploadBlobFile
+	transferManager  azblob.TransferManager
 }
 
 // New is the constructor for Ingestion.
 func New(db, table string, mgr *resources.Manager) (*Ingestion, error) {
```

Comment:
Is anyone using the old API?
If not consider merging

Reply:
Irrelevant now since we added options